components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -3181,6 +3182,31 @@ func TestCreateProject(t *testing.T) {
 			assert.Equal(t, storage_errors.ErrConflict, err)
 			assert.Nil(t, resp)
 		},
+		"does not create project if max number projects of allow surpassed": func(t *testing.T) {
```

Comment:
this test name is a little confusing. Maybe "does not create project if max number of projects allowed has been reached" ?